dali/server/daserver.cpp

```diff
@@ -616,8 +616,6 @@ int main(int argc, const char* argv[])
         mpServer->installWhiteListCallback(whiteListHandler);
 #ifndef _CONTAINERIZED
         setMsgLevel(fileMsgHandler, serverConfig->getPropInt("SDS/@msgLevel", 100));
-#else
-        setupContainerizedLogMsgHandler();
```

Comment:
Not related to the bug, but this function has already been called earlier in the function.